test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -65,6 +65,16 @@ var _ = framework.KubeDescribe("[HPA] Horizontal pod autoscaling (scale resource
 		})
 	})
 
+	framework.KubeDescribe("[Serial] [Slow] StatefulSet", func() {
```

Comment:
Note that the `SubresourceReference` in that proposal was later replaced with `ScaleTargetRef` in the code. It is basically just a way to tell the HPA which other controller you want it to autoscale, by giving it the group-version-kind + name. You shouldn't need any such references in your testing, because you already know what controller you intend to scale.